### DISCUSSION TITLE
Move Wiki to GitBook and auto deploy

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#281] [Ideas]
I am personally not a fan of the Wiki functionality in GitHub and would suggest to actually disable it in favor of a markdown based doc/ folder in the repo that is deployed using GitBook as I have done it in a very simple was [here](https://github.com/ASCS-eV/EVES). You can then link the documentation directly here with a web link:
![grafik](https://github.com/user-attachments/assets/6505b0f9-f531-49e6-9ae9-6e3c79124b14)

**Comment by aneuwald-ctw** [comment]
@jdsika thanks for your personal input. We already changed it to another repo, you can check it here: https://github.com/Lichtblick-Suite/docs
